docs/sources/operations/storage/table-manager.md

```diff
@@ -31,6 +31,7 @@ The Table Manager supports the following backends:
   - [Amazon DynamoDB](https://aws.amazon.com/dynamodb)
   - [Google Bigtable](https://cloud.google.com/bigtable)
   - [Apache Cassandra](https://cassandra.apache.org)
+  - [Amazon S3](https://aws.amazon.com/s3)
```

Comment:
The stores listed here are the only ones for which retention is enforced by Table Manager. The object stores not being supported for retention is mentioned below https://github.com/grafana/loki/pull/3538/files#diff-1d27edb4dce70c907414922910bbdebb8bdc82423a79f8c86efbf33d12482e0bR37